agent/index.js

```diff
@@ -141,6 +148,19 @@ class Agent {
 		});
 	}
 
+	_runOnce(Job) {
+		if (!runOnce) {
+			return;
+		}
+		const taskLogger = this.logger.child({
+			namespace: LOGGER_NAMESPACES.TASK,
+			job: Job.name,
+			uid: new Chance().guid(),
+		});
+		const job = new Job(this.codefreshAPI, this.kubernetesAPI, taskLogger);
```

Comment:
You should pass `this.runtimes`, not `kubernetesAPI`

Reply:
done